### PR TITLE
condensed activate script

### PR DESCRIPTION
**Pull request by iknight7000**
Co-authored-by: Chris <chris-chilton@hotmail.com>

# Pull Request Template

## Description

By condensing the deactivate/activate function to a single call we have eliminated wasteful code while creating a working script activation function that can be used for the Twitter bot as it improves. [[VIDEO LINK HERE](https://www.loom.com/share/c2744f501f37495d8b3c9717f55b9bc5)]

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

This bug was tested by activating a script locally. 

- [x] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by mikecarrier4 — APPROVED** [review]
Good job with the cleaning up of the script master class.  This part of the old code base was always a bit convoluted.

**Review by gaben3722 — APPROVED** [review]
Everything looks updated and cleaned. Approved.

File: app/bot.py [hunk 1]
```diff
@@ -2,20 +2,24 @@
...
 load_dotenv(find_dotenv())
 DB = Database()
 
-MAP_API = os.getenv("MAP_API") # Maps to Google Places API
+# Maps to Google Places API
+MAP_API = os.getenv("MAP_API")
 
-bot_name = os.getenv("BOT_NAME")  # Need bot name
+# Need bot name
+bot_name = os.getenv("BOT_NAME")
```
Comment (ChristopherKchilton) [inline app/bot.py:24]: Nice adjusting of comment location.

File: app/db.py [hunk 2]
```diff
@@ -164,23 +167,144 @@ class BotScripts(Base):
     convo_node = Column(Integer)
     use_count = Column(Integer)
     positive_count = Column(Integer)
-    success_rate = Column(Float)
+    # success_rate = Column(Float)
     active = Column(Boolean)
```
Comment (ChristopherKchilton) [inline app/db.py:171]: We need to correct the table so it holds success_rate

**Review by ChristopherKchilton — APPROVED** [review]
Removing an unneeded class ScriptMaster. It is a good refactor. I approve of this change.

**Pull request merged**